forc/SanityCheck/forcing_datm_sanity.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+                           #hyun-chul.lee@noaa.gov
+import matplotlib
+#matplotlib.use('agg')
+import matplotlib.pyplot as plt
+import matplotlib.colors as mcolors
+import netCDF4 as nc
+from netCDF4 import Dataset
+import numpy as np
+import argparse
+import glob
+import os
+import yaml
+import sys
+
+def plot_zonal_ave(data1zave,data1zmax,data1zmin,lats1,data2zave,data2zmax,data2zmin,lats2,plotpath,v,sdate,d1nm,d2nm,ne):
```

Comment:
... that's a long list of arguments! Consider maybe using at least simple data structures at least to clarify your code?